errors/error.go

```diff
@@ -162,6 +149,13 @@ func E(args ...interface{}) *Error {
 
 	errs := L()
 	for _, arg := range args {
+		if arg == nil {
+			panic("errors.E() called with nil value in argument")
+		}
+		// explicit check for deprecated usage of the API.
+		if reflect.TypeOf(arg).String() == "*config.Stack" {
+			panic("errors.E() does not support type *config.Stack anymore")
```

Comment:
I checked all the occurrences and this should never trigger, but it doesn't hurt to keep this panic just in case someone is using this type (it's public).

Reply:
lets add information that this is a bug and users should create an issue...

Reply:
I improved the panic message, now it prints:
```
panic: BUG: Deprecated use of argument type hcl.Range in the errors.E() function.
If this panic was triggered in the Terramate project, then please open an issue
at https://github.com/mineiros-io/terramate/issues.
```
wdyt?

Reply:
great ;) thanks